serving/installing-logging-metrics-traces.md

```diff
@@ -18,7 +16,7 @@ To clone the repository, run the following commands:
    git checkout v0.2.0
```

Comment:
Can we make this point to the release like the install doc does, instead of requiring the user to checkout the repo?

Reply:
Done